src/App.css

```diff
@@ -134,17 +147,32 @@ form > button {
 }
 
 .bookAuthor {
-  width: 17.813rem;
+  width: 20rem;
   height: 2.813rem;
   border-radius: 4px;
   border: solid 1px var(--white);
   background-color: #fff;
+  margin-right: 20px;
 }
 
 .bookTitle {
-  width: 20.5rem;
+  width: 40rem;
   height: 2.813rem;
   border-radius: 4px;
   border: solid 1px var(--white);
   background-color: #fff;
+  margin-right: 20px;
+}
+
+form {
+  display: flex;
```

Comment:
- [ ] You have used display: flex 9 times, which is considered bad CSS practice because you are repeating yourself a lot, I would recommend you create a class with display:flex on it and then use it on your HTML that you need 
![image](https://user-images.githubusercontent.com/62705947/222692279-d71e8ba4-0d34-4237-96b6-2524264cbc2e.png)